Homarus/README.md

```diff
@@ -13,14 +33,26 @@ Alias "/homarus" "/var/www/html/Crayfish/Homarus/src"
 </Directory>
 ```
 
-### Testing
-* Clone this microservice into `/var/www/html/Crayfish`
-* Install the depencies using composer `composer install`
-* Place the above confir in `/etc/apache2/conf-available` as `Homarus.conf`.  
-* Enable the conf: `sudo a2enconf Homarus`.  Restart Apache: `sudo systemctl restart apache2`
-* Create a test video by going here: `http://localhost:8080/fcrepo/rest/` and uploading a binary
-* Test via curl as below or using Postman
+This will put the Homarus at the /homarus endpoint on the webserver.
+
+## Configuration
+
+If your homarus installation is not on your path, then you can configure Homarus to use a specific executable by editing `executable` entry in [config.yaml](./cfg/config.example.yaml).
```

Comment:
s/homarus/`ffmpeg`

Reply:
I read that as you can use something other than `ffmpeg` to do your conversion?

Reply:
s/homarus/executable ?